workflows: add docs for Revive workflow (#3512)
AOSC-Dev/aosc-os-abbs: README.md

achieved by the "groups" defined in the `groups/` directory, with files defining
a list of packages to build.

Automation
----------

> Under GitHub issues and pull requests, one can directly invoke package building on GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.

Reviewer: You may initiate package build tests via GitHub Actions.

_One does not simply use "one."_

Author: ```suggestion
Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
```
